app/views/info/privacy.html.erb

```diff
@@ -4,7 +4,7 @@
   <p>We use Facebook to authenticate volunteer editors who edit listings on the site. </p>
   <p>The Facebook profile name of a volunteer editor is processed by storing it alongside any revisions they make to listings on the site. Their profile name is visible only to other editors on the site, and is used to help them collaborate with other editors and see who has made revisions. We do not store any other information.</p>
   <p>Users who wish to revoke permission for the <%= tc("site_name") %> to access this site can do so by clicking "Disable my login" when logged into the site. This link is shown in the top-right of the editors home page.</p>
-  <p>Users may request deletion of this data by sending an email to <%= tc("email_address") %> requesting deletion. </p> 
+  <p>Users may request deletion of this data by sending an email to <%= tc("email_address") %> requesting deletion. The user's data will be deleted the site administrator receives this request. </p> 
```

Comment:
@gonogo there seem to be some words missing. Not sure if you meant something like:
```suggestion
  <p>Users may request deletion of this data by sending an email to <%= tc("email_address") %> requesting deletion. The user's data will be deleted when the site administrator receives this request. </p> 
```